codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerInstantiator.kt

```diff
@@ -86,3 +91,38 @@ fun serverInstantiator(codegenContext: CodegenContext) =
         defaultsForRequiredFields = true,
         customizations = listOf(ServerAfterInstantiatingValueConstrainItIfNecessary(codegenContext)),
     )
+
+class ServerBuilderInstantiator(
+    private val symbolProvider: RustSymbolProvider,
+    private val symbolParseFn: (Shape) -> ReturnSymbolToParse,
+) :
+    BuilderInstantiator {
```

Comment:
```suggestion
) : BuilderInstantiator {
```

The current formatting looks odd.